### ISSUE TITLE
Multilingual document failure

### ISSUE THREAD

**Issue by junjieweb** [facebook/docusaurus#7531]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#reporting-new-issues).

### Description

Switch to any language, the document is displayed in English
![image](https://user-images.githubusercontent.com/90107547/170990077-bdd5e5eb-28fc-4268-8aed-75a7cb554392.png)

### Self-service

- [ ] I'd be willing to address this documentation request myself.

**Comment by junjieweb** [comment]
You're welcome! The search in the top right corner works fine.

**Comment by junjieweb** [comment]
> Thanks for the report! This is weird...

You're welcome! The search in the top right corner works fine.

**Comment by Josh-Cena** [comment]
The translation download failed: https://app.netlify.com/sites/docusaurus-2/deploys/62946916320aa8000a66ec21

It should be gone the next time we deploy

**Comment by slorber** [comment]
Should be fixed now

temporary error + also a translation error preventing the site to build

For some reasons the french site has a very weird missing stylesheet: https://docusaurus.io/fr/

Hope this is only temporary, triggering a rebuild with clean caches

**Comment by forresst** [comment]
> 
> temporary error not failing the build (now reported to Crowdin) + also a translation error preventing the site to build
> 

Sorry for the translation error, it's me who is at fault.

**Comment by slorber** [comment]
No problem @forresst :D that's already awesome that you always keep up with the French translations 💯 we can only thank you for that

**Comment by slorber** [comment]
Don't know how to fix it, but this must be a Netlify bug

- Ok: https://62974eced5fd1700090247a9--docusaurus-2.netlify.app/fr/assets/css/styles.30d28e71.css
- Fail: https://docusaurus.io/fr/assets/css/styles.30d28e71.css

Yet it's the exact same deployment

**Comment by slorber** [comment]
https://docusaurus.io/fr/ doesn't look normal to me, only the stylesheet file can't load for some reason 🤷‍♂️ wonder if I'm alone

**Comment by slorber** [comment]
![Uploading CleanShot 2022-06-01 at 15.07.33.png…]()

🤪  probably a local CDN issue hopefully

**Comment by Josh-Cena** [comment]
Ah, should be a CDN issue. It's all good on my side—the deployment is fine.

**Comment by Josh-Cena** [comment]
Going to close as resolved

**Comment by slorber** [comment]
Yes 👍 

Still not fixed for me (and other people in France) but I've sent a ticket to Netlify and will track if they fix it

**Comment by forresst** [comment]
For information, the problem is still not solved for the French version.  @Josh-Cena Can you please browse the French version, to confirm that it still works for you?

**Comment by Josh-Cena** [comment]
Yep, still works for me. I'm behind a VPN so at least I can confirm that it works in the US and China😄 There's no French proxy so the closest I can get is that it _seems_ to work in Germany and Netherlands. (Assuming the CDN server is decided based on my IP and my IP is successfully masked by the proxy server!)

**Comment by slorber** [comment]
> For information, the problem is still not solved for the French version. @Josh-Cena Can you please browse the French version, to confirm that it still works for you?

Yes @forresst 

I have opened a Netlify ticket with a network HAR file + netlify request ids etc... Can be reproduced with curl. Clearly this is on their side.

Edit: Oh, now it seems to work for me! Still broken for you?

**Comment by forresst** [comment]
Now it's ok for me too

**Comment by slorber** [comment]
Great 👌 

Totally forgot but we have Cloudflare on top on Netlify

Apparently this specific issue was more related to Cloudflare than Netlify according to their support

<img width="662" alt="CleanShot 2022-06-04 at 17 49 58@2x" src="https://user-images.githubusercontent.com/749374/172013591-31059738-9cbb-4977-b9f9-fd5e2de4c163.png">
